package.json

```diff
@@ -39,7 +39,7 @@
     "@react-navigation/native": "6.1.18",
     "@react-navigation/stack": "6.4.1",
     "@rneui/base": "4.0.0-rc.8",
-    "@sentry/react-native": "5.27.0",
```

Comment:
The package upgrade is because errors, probably related to iOS 18.

Reply:
Xcode 16 :)

Reply:
Changing this needs to ipdate the cocoaSDK version on the xcode project as well

Reply:
Make sure that version `v8.36.0` is used in the workspace Package.resolve of the xcode project

![image](https://github.com/user-attachments/assets/3878a368-ab1a-4861-8d42-d8b32109be1b)